packages/LablePicture/package/NewLablePicture/index.tsx

```diff
@@ -0,0 +1,344 @@
+import styled, { keyframes } from 'styled-components';
+import React, { useState, useRef } from 'react';
+import styles from './style.module.css';
+
+const Button: React.FC<{ isEditMode: boolean }> = ({ isEditMode }) => {
+  const [imageUrl, setImageUrl] = useState<string | null>(null);
+  const fileInputRef = useRef<HTMLInputElement>(null); // Ref for the file input
+
+  // Handle file selection
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0];
+    if (file) {
+      const reader = new FileReader();
+      reader.onload = (e) => {
+        if (e.target?.result) {
+          setImageUrl(e.target.result as string); // Set the image URL
+        }
+      };
+      reader.readAsDataURL(file); // Read the file as a data URL
+    }
+  };
+
+  // Trigger file input dialog when the button is clicked
+  const handleButtonClick = () => {
+    if (fileInputRef.current) {
+      fileInputRef.current.click(); // Programmatically click the file input
+    }
+  };
+
+  return (
+    <StyledWrapper>
+      {/* File input (hidden) */}
+      <input
```

Comment:
do not add file uploader like this refer to flashcard component how the upload button works you have to reuse that component